javathreads/src/main/java/bootcamp/Threads.java

```diff
@@ -0,0 +1,84 @@
+package bootcamp;
+
+import java.util.UUID;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Threads {
+
+
+    public static void main(String[] args) {
+
+        // Calling the procedure couple of times
+        timeConsuming(5);
```

Comment:
Idea is:
print 1
time consuming
print a
time consuming
print 2
time consuming
print b
time consuming
...
print 5
time consuming
print e
time consuming
Then it will be split between two threads , hence will execute 2 times faster